test/blackbox/BlackboxTestsLivelinessQos.cpp

```diff
@@ -134,13 +134,13 @@ TEST(LivelinessQos, ShortLiveliness_ManualByParticipant_Reliable)
     auto data = default_helloworld_data_generator(num_samples);
     reader.startReception(data);
 
-    size_t count = 0;
+    unsigned int count = 0;
     for (auto data_sample : data)
     {
         writer.send_sample(data_sample);
-        ++count;
-        reader.block_for_at_least(count);
-        std::this_thread::sleep_for(std::chrono::milliseconds(writer_sleep_ms));
+        reader.wait_liveliness_recovered(count + 1);
+        reader.wait_liveliness_lost(count + 1);
+        count++;
```

Comment:
Why not keep `++count;` at the beginning and call the methods just with `count`?

``` cpp
++count;
reader.wait_liveliness_recovered(count);
reader.wait_liveliness_lost(count);
```

Reply:
:+1: Changed